modules/kit-elements-defaults/assets/js/editor/commands/confirm-creation.js

```diff
@@ -0,0 +1,90 @@
+export default class ConfirmCreation extends $e.modules.editor.CommandContainerBase {
+	static introduction;
+	static #introductionKey = 'kit_elements_defaults_create_dialog';
+
+	validateArgs() {
+		this.requireContainer();
+	}
+
+	async apply( { container } ) {
+		if ( ! this.constructor.introduction ) {
```

Comment:
Try to extract all `introduction` related code to an external class/function.
Then the command can be something like: 
```
import confirmDialog ...

apply( args ) {
  if ( ! confirmDialog.viewed ) {
    confirmDialog.show(); // on confirm - re-run this command.
    return; 
  }
  
  $e.run( 'kit-elements-defaults/create', args );
}

Reply:
Done